applications/MeshMovingApplication/python_scripts/ale_fluid_solver.py

```diff
@@ -100,10 +110,12 @@ def Initialize(self):
         # Saving the ALE-interface-parts for later
         # this has to be done AFTER reading the ModelPart
         self.ale_boundary_parts = []
-        main_model_part_name = self.settings["model_part_name"].GetString()
+        main_model_part_name = self.settings["fluid_solver_settings"]["model_part_name"].GetString()
+
+        ale_boudary_parts_params = self.settings["ale_boundary_parts"]
 
-        for i_name in range(self.ale_boudary_parts_params.size()):
-            sub_model_part_name = self.ale_boudary_parts_params[i_name].GetString()
+        for i_name in range(ale_boudary_parts_params.size()):
```

Comment:
In python we normally avoid `for i in range(len(v)):` and just do `for x in v:`. Is there a reason you cannot write the following?

```
for param in ale_boundary_parts_params:
    ...
```